aws/required-permissions.md

```diff
@@ -27,105 +27,186 @@ Check out the templates directly for more information.
+| `kms:GenerateDataKey` | Action | **Optional.** Generates a Data key used to encrypt the scan result messages sent to `ScanResultTopic` or the error messages sent to encrypted `ScannerDLQ` and `PostScanActionDLQ` |
+| `lambda:CreateAlias` | Action | Creates an alias for Lambda functions created by Cloud One File Storage Security |
+| `lambda:DeleteAlias` | Action | Deletes the alias of Lambda functions created by Cloud One File Storage Security |
+| `lambda:GetFunctionConfiguration` | Action | Retrieves the configuration of Lambda functions created by Cloud One File Storage Security |
+| `lambda:GetLayerVersion` | Action | Retrieves the information of a version of the Lambda layers created or managed by Cloud One File Storage Security |
+| `lambda:InvokeFunction` | Action | Lets AWS service `EventBridge` and `SNS` invoke the Lambda functions created by Cloud One File Storage Security |
+| `lambda:PublishVersion` | Action | Publishes a new version of the Lambda functions created by Cloud One File Storage Security |
+| `lambda:UpdateAlias` | Action | Updates the alias of Lambda functions created by Cloud One File Storage Security |
+| `lambda:UpdateFunctionConfiguration` | Action | Updates the configuration of Lambda functions created by Cloud One File Storage Security |
+| `organizations:DescribeOrganization` | Action | Retrieves information about the organization that the current AWS account belongs to |
+| `sns:Publish` | Action | Publishes a message to a `ScanResultTopic` created by Cloud One File Storage Security |
+| `sqs:DeleteMessage` | Action | Deletes messages from the `ScannerDLQ` and `PostScanActionDLQ` created by Cloud One File Storage Security |
+| `sqs:GetQueueAttributes` | Action | Gets the attributes of the `ScannerDLQ` and `PostScanActionDLQ` created by Cloud One File Storage Security |
+| `sqs:ReceiveMessage` | Action | Receives messages from the `ScannerDLQ` and `PostScanActionDLQ` created by Cloud One File Storage Security |
+| `sqs:SendMessage` | Action | Sends scan messages to the `ScannerDLQ` and `PostScanActionDLQ` created by Cloud One File Storage Security |
+| `ssm:GetParameter` | Action | Retrieves values in AWS Systems Manager that Cloud One File Storage Security created |
+| `sts:AssumeRole` | Action | Grants to different principals the permission to switch roles created by Cloud One File Storage Security |
+| `s3:DeleteObject` | Action | Removes the malicious object from the scanning bucket after being quarantined |
+| `s3:GetObject` | Action | Retrieves the object to be scanned or quarantined from Amazon S3 |
+| `s3:GetObjectTagging` | Action | Lists the tags of the scanned object |
+| `s3:PutObject` | Action | Puts the malicious object in the quarantine bucket |
+| `s3:PutObjectTagging` | Action | Puts tags on the scanned object |
+
+### Permissions required by ManagementRole
```

Comment:
The ManagementRole's permissions should be covered in above sections. Why would we need this section?

Reply:
I would like to put ManagementRole's permissions in the independent section is that these permissions are used by **Cloud One File Storage Security** to manage the stack. 
The rest of the permissions in `### Permissions required by account scanner stack` are used by the **resources** to scan/ tag/ quarantine... and so on. 

External users/ customers might more care about the ManagementRole's permissions, so I put it into the independent section.

Reply:
Got it. I'm confused because only account scanner's is separated, but it's okay for now.